packages/x-components/src/x-modules/search/components/sort-dropdown.vue

```diff
@@ -76,7 +76,7 @@
 </script>
 
 <docs lang="mdx">
-# Sort Dropdown
+## Sort Dropdown
```

Comment:
Any reason for having a header with the component's name here? When the docs are generated this would result in a page with two headers with the same text (the main one and this one) shouldn't this be moved under examples or is there a specific reason?